### PR TITLE
Nav2 chrome

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] spaces on top of buttons
- [x] larger buttons in small mode
- [x] badges placement for big/small mode
- [x] nav menu
 - [x] avatar
 - [x] full name (needs plumbing from core, ticketed, you have to go to profile first)
 - [x] settings link
 - [x] help link
 - [x] signout link

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by chrisnojima** [comment]
cc: @cecileboucheron @adamjspooner

**Comment by adamjspooner** [comment]
This is looking great!

A few small things:

- This menu should be 8px away from the left edge. Screen Shot 2019-03-18 at 14 29 22
- This arrow should be closer the the avatar. Screen Shot 2019-03-18 at 14 33 10
- I think this is meant to overlap a bit more (@cecileboucheron?). Screen Shot 2019-03-18 at 14 34 46
- There's a small bug with the tooltips. Moving over the current tooltip (moving your cursor up quickly) highlights the correct tab, but it doesn't show that tab's tooltip. tooltips

**Comment by chrisnojima** [comment]
@adamjspooner fixed

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/router-v2/tab-bar/container.js [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 import {memoize} from '../../util/memoize'
+import openURL from '../../util/open-url'
 
 type OwnProps = {|
   selectedTab: Tabs.Tab,
 |}
 
 const mapStateToProps = state => ({
   _badgeNumbers: state.notifications.navBadges,
+  fullname: TrackerConstants.getDetails(state, state.config.username).fullname || '',
```
Comment (buoyad) [inline shared/router-v2/tab-bar/container.js]: don't need `|| ''`

File: shared/router-v2/tab-bar/index.desktop.js [hunk 2]
```diff
@@ -31,75 +31,136 @@ const tabs = [
...
+            <Kb.Box2 direction="horizontal" style={styles.osButtons} fullWidth={true} />
+            <Kb.ClickableBox onClick={this._showMenu}>
               <Kb.Box2
                 direction="horizontal"
+                gap="tiny"
                 fullWidth={true}
-                className={t === p.selectedTab ? 'tab-selected' : 'tab'}
-                style={styles.tab}
+                style={styles.nameContainer}
+                alignItems="center"
```
Comment (buoyad) [inline shared/router-v2/tab-bar/index.desktop.js]: `centerChildren={true}` to line this up
image

Comment (chrisnojima) [reply shared/router-v2/tab-bar/index.desktop.js]: tnx

File: shared/router-v2/tab-bar/container.js (hunk 1 above)
Comment (chrisnojima) [reply shared/router-v2/tab-bar/container.js]: this is needed actually as the type in detail is nullable

Comment (buoyad) [reply shared/router-v2/tab-bar/container.js]: ah ok :+1:

**Pull request merged**